tests/api/test_groups.py

```diff
@@ -272,13 +280,13 @@ def test_GroupsMixin_link_concession_group_funding_source_expiry(
     mock_ClientProtocol_post_link_concession_group_funding_source, mocker
 ):
     client = GroupsMixin()
-    mocker.patch.object(client, "_format_concession_expiry", return_value="formatted concession expiry")
+    mocker.patch.object(client, "_format_expiry", return_value="formatted concession expiry")
```

Comment:
Just a nit but we might as well get rid of the word `concession` here too

Reply:
Removed in https://github.com/cal-itp/littlepay/pull/45/commits/484cf53ab421a1254e28da48bb39364e141b8f8c